<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSRemoteSettingsDialog</class>
 <widget class="QWidget" name="OBSRemoteSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QToolButton" name="AccesListBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>33</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>101</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="title">
    <string notr="true"/>
   </property>
   <widget class="QCheckBox" name="ServerEnableChBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>95</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DNSDEnableChBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>95</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>DNS-SD</string>
    </property>
   </widget>
   <widget class="QComboBox" name="InterfacesComBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>261</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="OkBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>95</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>95</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
